operator/operator.go

```diff
@@ -284,13 +284,35 @@ func (o *Operator) Start(ctx context.Context) error {
 			go o.aggregatorRpcClient.SendSignedStateRootUpdateToAggregator(&signedStateRootUpdateMessage)
 			continue
 
-		case checkpointTaskResponse, ok := <-checkpointTaskResponseChan:
+		case checkpointTaskCreatedEvent, ok := <-checkpointTaskCreatedChan:
 			if !ok {
 				o.logger.Info("Closing Operator")
 				return o.Close()
 			}
 
 			o.metrics.IncNumTasksReceived()
+
+			var checkpointMessages messages.CheckpointMessages
```

Comment:
Let's move this to a separate function and call as goroutine since it has GetAggregatedCheckpointMessages request on which the whole loop would wait for response

Reply:
Do you think it's that problematic? Considering checkpoints are done once every some hours and this should ideally not take too much time, I think we could just do it like this and avoid one extra channel. Wdyt?

Reply:
Either way, the error management there is pretty simple, so making it a goroutine is simple as well

Reply:
Done in [dae24b9](https://github.com/NethermindEth/near-sffl/pull/57/commits/dae24b966e94463a4f15273e5ae1074a0807385c)